src/modules/NotFoundPage/NotFoundPage.module.scss

```diff
@@ -0,0 +1,97 @@
+@import '../../styles/utils/variables/colors';
+
+@mixin onTablet {
+  @media (max-width: 640px) {
+    @content;
+  }
+}
+
+@mixin onDesktop {
+  @media (min-width: 1200px) {
+    @content;
+  }
+}
+
+$footerAndHeaderHeight: 144px;
+$footerAndHeaderHeightTablet: 305px;
+$footerAndHeaderHeightDesktop: 160px;
+
+.container {
+  height: calc(100vh - #{$footerAndHeaderHeight});
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+
+  @include onTablet {
+    height: calc(100vh - #{$footerAndHeaderHeightTablet});
+  }
+
+  @include onDesktop {
+    height: calc(100vh - #{$footerAndHeaderHeightDesktop});
+  }
+
+}
+
+.not_found_logo {
+  height: 160px;
+}
+
+.error_message {
+  color: $color__primary;
+
+  &_dark {
+    color: $color__dark-theme__white;
+  }
+}
+
+.subtitle {
+  color: $color__primary;
+
+  &_dark {
+    color: $color__dark-theme__white;
+  }
+}
+
+.headline {
+  font-size: 28px;
+  &_dark {
+    color: $color__dark-theme__white;
+  }
+}
+
+.button {
+  margin-top: 12px;
+  cursor: pointer;
+  padding: 15px;
+  border-radius: 15px;
+  border: 2px solid #3B3E4A;
+  transition: scale 0.05s ease;
+  border-radius: 12px;
+  font-size: 18px;
+  text-decoration: none;
+  color: $color__primary;
+
+  &:hover {
+    scale: 1.1;
+  }
+
+  &_dark {
+  margin-top: 12px;
+  cursor: pointer;
+  padding: 15px;
+  border-radius: 15px;
+  border: 2px solid $color__dark-theme__white;
+  transition: scale 0.05s ease;
+  border-radius: 12px;
+  font-size: 18px;
+  text-decoration: none;
+  color: $color__dark-theme__white;
```

Comment:
We already have a ready-made mixin with all the typography from the figma, use it whenever possible
`src/styles/utils/mixins/`